shared/src/validation-schemas/sign-in/sign-in.schema.ts

```diff
@@ -0,0 +1,11 @@
+import * as Joi from 'joi';
+import { UserSignInPayload } from '../../common/types/types';
+
+const SigninSchema = Joi.object<UserSignInPayload>({
+  email: Joi.string()
```

Comment:
Create enum for:
rules - SignInValidationRule
messages: SignInValidationMessage
in the `shared/src/common/enums/validation/`